modules/data.atmosphere/R/metgapfill.R

```diff
@@ -15,113 +15,128 @@
 ##' @param lst is timezone offset from UTC, if timezone is available in time:units atribute in file, it will use that, default is to assume UTC
 ##' @author Ankur Desai
 ##'
-metgapfill <- function(in.path, in.prefix, outfolder, start_date, end_date, lst=0, overwrite=FALSE, verbose=FALSE, ...){
-
-  library(REddyProc)    
-  #REddyProc installed to ~/R/library by install.packages("REddyProc", repos="http://R-Forge.R-project.org", type="source")
-  #dependency minpack.lm may not install automatically, so install it first
+metgapfill <- function(in.path, in.prefix, outfolder, start_date, end_date, lst = 0,
+                       overwrite = FALSE, verbose = FALSE, ...) {
+  
+  library(REddyProc)
+  # REddyProc installed to ~/R/library by install.packages('REddyProc',
+  # repos='http://R-Forge.R-project.org', type='source') dependency minpack.lm may not install
+  # automatically, so install it first
   library(ncdf4)
   library(udunits2)
   library(PEcAn.utils)
   library(lubridate)
   
   # get start/end year code works on whole years only
   start_year <- year(start_date)
-  end_year <- year(end_date)
-
-  if(!file.exists(outfolder)){
+  end_year  <- year(end_date)
+  
+  if (!file.exists(outfolder)) {
     dir.create(outfolder)
   }
   
   rows <- end_year - start_year + 1
-  results <- data.frame(file=character(rows), host=character(rows),
-                        mimetype=character(rows), formatname=character(rows),
-                        startdate=character(rows), enddate=character(rows),
-                        dbfile.name = in.prefix,
+  results <- data.frame(file = character(rows), 
```

Comment:
I much prefer one variable per line. did formatR do this? Because (I think) it did the opposite over starting with `newmet <- met[` on line 111 here: https://github.com/bpbond/pecan/pull/2/files#diff-d0ac9657ee0b984a446bea24ac1b77d3R82